src/routes/pricing/faq.svelte

```diff
@@ -8,36 +8,44 @@
         answer: string;
     }[] = [
         {
-            question: 'When will Appwrite’s paid plans be available?',
-            answer: 'It is our aim to release pricing in the midst of Q4 2023. Please do note that this timeline is subject to many factors and, therefore, not set in stone. Everyone with an Appwrite Cloud account will receive a notification when our paid plans are available.'
-        },
-        {
-            question: 'Can I still self-host Appwrite for free?',
-            answer: 'Yes! Appwrite Self Hosted will always be Open Source and free for you to use. You can find the latest version of Appwrite on our GitHub repository.'
-        },
-        {
-            question: 'What will happen to my current account when pricing is available?',
-            answer: 'All current Cloud users will be notified about the pricing plans being active. Depending on your situation, you will either need to upgrade your account to a Pro plan, or you can happily continue on a Starter plan. You will receive an email once this occurs.'
+            question: 'Can I self-host Appwrite for free?',
+            answer: 'Yes! Appwrite Self Hosted will always be Open Source and free for you to use, without any limits. You can find the latest version of Appwrite on our GitHub repository.'
         },
         {
             question: 'Does Appwrite have a trial period?',
-            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can choose to either subscribe to the Pro plan or continue on a Starter plan.'
+            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can either subscribe to the Pro plan or continue on a Starter plan.'
         },
         {
             question: 'What payment methods does Appwrite support?',
-            answer: 'Appwrite currently supports credit and debit card payments. We will be working to constantly add support for more methods. Please reach out to us in case this is an issue for you.'
+            answer: 'Appwrite currently supports credit and debit card payments. We will be working to add support for more methods constantly. Please reach out to us in case this is an issue for you.'
```

Comment:
@LauraDuRy do we also want which card brands we allow, (Visa, Mastercard, Amex, Diners Club, etc.)?

Reply:
Not here in the fAQ, but we would like to the docs. We will do this once docs is live.

Reply:
alright 👍